smart_open/s3.py

```diff
@@ -28,6 +28,10 @@
 MODES = (READ, READ_BINARY, WRITE, WRITE_BINARY)
 """Allowed I/O modes for working with S3."""
 
+BINARY_NEWLINE = b'\n'
+TEXT_NEWLINE = b'\n'
```

Comment:
Both are binary?

Reply:
You're right. We're not using TEXT_NEWLINE right now, so I removed it.